### PR TITLE
Add missing device name tags

### PR DESCRIPTION
**Pull request by kbogtob**
### What does this PR do?

It adds the new `device_name` tag on places that produced `device` tags.

### Motivation

Homogeneization of devices tags.

### Additional Notes

I tried to be exhaustive but I'm not 100% sure. It should so help me if it is not.

### Describe your test plan

`device_name` tags are produced on docker/containerd/network checks. I'm not sure how to test on series.

### CODE REVIEW

**Review by albertvaka — DISMISSED** [review]

**Comment by albertvaka** [comment]
LGTM once tests are green.

File: releasenotes/notes/add-missing-device_name-tags-07e712f31b5a774d.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - Add missing `device_name` tags on docker, containerd et network checks.
```
Comment (remeh) [inline releasenotes/notes/add-missing-device_name-tags-07e712f31b5a774d.yaml:10]: Nit: the format of `.rst` files is not markdown, please see release notes entries (on top of my mind but I'm not a specialist: two quotes for "code escaping" and no ` -` to enumerate)
`et` → `and`

File: pkg/collector/corechecks/containers/docker.go [hunk 2]
```diff
@@ -392,7 +394,9 @@ func (d *DockerCheck) reportIOMetrics(io *cmetrics.ContainerIOStats, tags []stri
 	// Write values per device, or fallback to sum
 	if len(io.DeviceWriteBytes) > 0 {
 		for dev, value := range io.DeviceWriteBytes {
-			sender.Rate("docker.io.write_bytes", float64(value), "", append(tags, "device:"+dev))
+			devTags := append(tags, "device:"+dev)
```
Comment (remeh) [inline pkg/collector/corechecks/containers/docker.go:397]: Nit: you could have been with just adding the new field to the append call:

`append(tags, "device:"+dev, "device_name:"+dev)`

**Review by remeh — DISMISSED** [review]
Hey, left a few cosmetic comments. The code change looks good to me.

**Review by ahmed-mez — DISMISSED** [review]
LGTM, +1 for @remeh 's comments

**Review by albertvaka — APPROVED** [review]

**Pull request merged**